### PR TITLE
configure spring boot, added Book entity with service  and repository…

### PR DESCRIPTION
**Pull request by Nonzeq**
… implementation

### CODE REVIEW

File: src/main/java/com/kobylchak/bookstore/repository/BookRepositoryImpl.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+
+    private SessionFactory sessionFactory;
+
+    @Autowired
+    public BookRepositoryImpl(SessionFactory sessionFactory) {
+        this.sessionFactory = sessionFactory;
+    }
```
Comment (MentorMA) [inline src/main/java/com/kobylchak/bookstore/repository/BookRepositoryImpl.java]: Lets use lombok instad of this approach

Comment (MentorMA) [reply src/main/java/com/kobylchak/bookstore/repository/BookRepositoryImpl.java]: @requiredArgsConstructor

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/com/kobylchak/bookstore/BookStoreApplication.java [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class BookStoreApplication {
+    @Autowired
+    private BookService bookService;
```
Comment (Rommelua) [inline src/main/java/com/kobylchak/bookstore/BookStoreApplication.java:15]: Lets use lombok for constructor creating
```suggestion
    private BookService bookService;
```

File: src/main/java/com/kobylchak/bookstore/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```
Comment (Rommelua) [inline src/main/java/com/kobylchak/bookstore/model/Book.java]: do not use Data with entity classes

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Pull request closed without merging**